src/client/components/pages/registration-auth.js

```diff
@@ -23,6 +23,11 @@ const Button = require('react-bootstrap').Button;
 const PageHeader = require('react-bootstrap').PageHeader;
 const Alert = require('react-bootstrap').Alert;
 
+/**
+* Renders the document to sign up as an editor of Bookbrainz
```

Comment:
Hmm, that's not exactly what this page does. Have a look on the site and see if you can write a more accurate description: https://www.bookbrainz.org/register (hint: we use MusicBrainz accounts to sign in to the site!)